spec/controllers/equipment_models_controller_spec.rb

```diff
@@ -1,421 +1,273 @@
-                                                             name: nil)
-          end.not_to change(EquipmentModel, :count)
-        end
-        it { is_expected.to render_template(:new) }
+      let!(:missed) do
+        FactoryGirl.create(:missed_reservation, equipment_model: model)
       end
-    end
-    context 'with non-admin user' do
-      before { sign_in FactoryGirl.create(:user) }
-      it 'should redirect to root' do
-        post :create,
-             equipment_model: FactoryGirl.attributes_for(:equipment_model)
-        expect(response).to redirect_to(root_url)
+      let!(:starts_today) do
+        FactoryGirl.create(:reservation, equipment_model: model,
+                                         start_date: Time.zone.today,
+                                         due_date: Time.zone.today + 2.days)
       end
-    end
-  end
-
-  describe 'PUT update' do
-    context 'with admin user' do
-      before { sign_in FactoryGirl.create(:admin) }
-      context 'with valid attributes' do
-        before do
-          put :update, id: model, equipment_model:
-          FactoryGirl.attributes_for(:equipment_model, name: 'Mod')
-        end
-        it { is_expected.to set_flash }
-        it 'sets @equipment_model to selected model' do
-          expect(assigns(:equipment_model)).to eq(model)
-        end
-        it 'updates attributes' do
-          model.reload
-          expect(model.name).to eq('Mod')
-        end
-        it { is_expected.to redirect_to(model) }
+      let!(:starts_this_week) do
+        FactoryGirl.create(:reservation, equipment_model: model,
+                                         start_date: Time.zone.today + 2.days,
+                                         due_date: Time.zone.today + 4.days)
       end
-
-      context 'without valid attributes' do
-        before do
-          put :update, id: model, equipment_model:
-          FactoryGirl.attributes_for(:equipment_model, name: nil)
-        end
-        it { is_expected.not_to set_flash }
-        it 'should not update attributes' do
-          model.reload
-          expect(model.name).not_to be_nil
-        end
-        it { is_expected.to render_template(:edit) }
+      let!(:starts_next_week) do
+        FactoryGirl.create(:reservation, equipment_model: model,
+                                         start_date: Time.zone.today + 9.days,
+                                         due_date: Time.zone.today + 11.days)
       end
-      it 'calls delete_files'
-    end
-    context 'with non-admin user' do
-      before { sign_in FactoryGirl.create(:user) }
-      it 'should redirect to root' do
-        put :update,
-            id: model,
-            equipment_model: FactoryGirl.attributes_for(:equipment_model)
-        expect(response).to redirect_to(root_url)
+      it 'includes @pending reservations' do
+        get :show, id: model
+        expect(assigns(:pending)).to \
```

Comment:
Same here re: `assigns` - I feel like we can just mock out the queries and test those messages.